server/game/gameSystems/GameSystemLibrary.ts

```diff
@@ -73,7 +73,7 @@ export function attachUpgrade<TContext extends AbilityContext = AbilityContext>(
 export function attack<TContext extends AbilityContext = AbilityContext>(propertyFactory: PropsFactory<IInitiateAttackProperties, TContext> = {}): CardTargetSystem<TContext> {
     return new InitiateAttackSystem<TContext>(propertyFactory);
 }
-export function capture<TContext extends AbilityContext = AbilityContext>(propertyFactory: PropsFactory<ICaptureProperties, TContext> = {}): GameSystem<TContext> {
+export function capture<TContext extends AbilityContext = AbilityContext>(propertyFactory: PropsFactory<ICaptureProperties, TContext> = {}): CardTargetSystem<TContext> {
```

Comment:
Actually you can just remove the return type completely and let it infer, there's no benefit to having it like this and it causes issues like the one you're presumably running into. I have a task to come in and just remove all of them in here but haven't gotten around to it yet, might just knock it out after this PR is merged.